instrumentation/spring/spring-boot-autoconfigure/src/test/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfigurationTest.java

```diff
@@ -60,7 +67,8 @@ void initializeProvidersAndOpenTelemetry() {
                 assertThat(context)
                     .hasBean("openTelemetry")
                     .hasBean("sdkTracerProvider")
-                    .hasBean("sdkMeterProvider"));
+                    .hasBean("sdkMeterProvider")
+                    .hasBean("sdkLoggerProvider"));
```

Comment:
Can you also add assertions on `sdkLoggerProvider` to the other tests in this class? (i.e. same as other signal providers)